modules/boss/src/main/ts/ephox/boss/api/DomUniverse.ts

```diff
@@ -48,6 +48,14 @@ export default (): Universe<SugarElement, Document> => {
     ], tag);
   };
 
+  const getLanguage = (element: SugarElement<Node>): Optional<string> => {
+    if (SugarNode.isElement(element)) {
+      return Attribute.getOpt(element, 'lang');
+    } else {
+      return Optional.none();
+    }
+  };
```

Comment:
```suggestion
  const getLanguage = (element: SugarElement<Node>): Optional<string> =>
    SugarNode.isElement(element) ? Attribute.getOpt(element, 'lang') : Optional.none();
```
I agree with Diogo, so just making the actual suggestion here.